package.json

```diff
@@ -49,6 +50,8 @@
   "devDependencies": {
     "chai": "^4.2.0",
     "chai-spies": "^1.0.0",
+    "eslint": "^7.4.0",
+    "eslint-plugin-es": "^3.0.1",
     "jsdoc": "^3.6.3",
     "mocha": "^7.1.0",
     "webcrypto": "git+https://github.com/nodejs/webcrypto.git",
```

Comment:
I would recommend adding [husky](https://www.npmjs.com/package/husky) as well for pre-commit hooks.